src/Console/Executor/FileHyphenationExecutor.php

```diff
@@ -9,21 +9,33 @@
 
 class FileHyphenationExecutor implements ExecutorInterface
 {
+    private ?string $word;
```

Comment:
Forgot to set default value to null? 

My suggestion would be not allow null and check if set (`isset` method) where needed (then of course do same with the other class)

And then finally instead of setting to null just use unset method